resources/js/views/meetings/Index.vue

```diff
@@ -81,7 +81,7 @@
       </template>
 
       <template v-slot:cell(start)="data">
-       {{ $date.utc(data.item.start).local().format('DD.MM.YY HH:mm') }}
+       {{ $date.utc(data.item.start).tz(userTimezone).format('DD.MM.YY HH:mm') }}
```

Comment:
We should use the locale date format with [$d](https://kazupon.github.io/vue-i18n/guide/datetime.html)!